spec/search_builder/sufia/my_shares_search_builder_spec.rb

```diff
@@ -22,4 +24,28 @@
                            "{!terms f=has_model_ssim}GenericWork,Collection",
                            "-depositor"]
   end
+
+  describe "mediated deposit" do
+    before do
+      allow(Flipflop).to receive(:enable_mediated_deposit?).and_return(mediation_enabled)
+    end
+
+    context "with mediated deposit enabled" do
+      let(:user_query) { nil }
+      let(:mediation_enabled) { true }
+      it "does includes suppressed switch" do
+        builder.show_only_shared_files(solr_params)
```

Comment:
Should this also live in the parent `before` block since it's common to both new specs?